### PR TITLE
Possible migration fix

### PR DESCRIPTION
**Pull request by toresbe**
Django seems to struggle with migrations in this case. Renaming the model to a temporary name seems to update the ForeignKey references, while dropping the table alone does not. This does not appear to disrupt the migration chain - apart from making it work.

### CODE REVIEW

**Comment by toresbe** [comment]
Note that the same trick actually does need to be applied to User as there is for some reason a reference made by the admin class which is not updated. However, I've worked that into the update to django2 branch

File: packages/fkweb/fk/forms.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class UserChangeForm(forms.ModelForm):
 
     class Meta:
         model = User
-        fields = ('email', 'password', 'date_of_birth', 'is_active', 'is_superuser')
+        fields = ('password', 'date_of_birth', 'is_active', 'is_superuser')
 
     def clean_password(self):
         # Regardless of what the user provides, return the initial value.
```
Comment (odinho) [inline packages/fkweb/fk/forms.py]: Could probably just do the same here for email. To make it readonly. And also set `email = forms.CharField(readonly=True)` or however it was. -- But this is very fine for now. We're not using this much. And might not for a whole lot longer. :P

File: packages/fkweb/fk/migrations/0002_rename_all_custom_tables_to_default.py [hunk 2]
```diff
@@ -24,20 +30,22 @@ class Migration(migrations.Migration):
             name='fileformat',
             table=None,
         ),
+        # By calling RenameModel, for some reason Django no longer
+        # fails to update the ForeignKey references in the schema.
+        migrations.RenameModel('Organization', 'OrganizationNew'),
         migrations.AlterModelTable(
-            name='organization',
```
Comment (odinho) [inline packages/fkweb/fk/migrations/0002_rename_all_custom_tables_to_default.py]: This migration has already been run for the servers. You should make a new migration with all this in it, a 0006 if you will.

That one will also be run on the production servers then.

You can test by pushing to dev. -- Though if it fails, you have to clean the database up manual...yyy. Maybe not actually. Postgresql has transactions on migrations too. So should be okay. Anyway, easy to test on dev, and put in the new database if it went bad.

**Comment by toresbe** [comment]
Closed because superseded by #180

**Pull request closed without merging**